GOVERNANCE.md

```diff
@@ -103,6 +103,26 @@ These members will be elected for a term of 1 year as follows:
      candidates will be the winners, 
    * all but one of the candidates that are tied for the number of votes choose to conceed.
 
+## Approving and Onboarding Regular Members
+
+Regular members can self-nominate by opening a PR to add themselves to the Regular
+member list in the [README.md](https://github.com/openjs-foundation/cross-project-council/blob/master/README.md). 
+The PR should include an indication of which project they have been active in for
+3 months as per the [CPC charter](https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#regular-members).
+
+The PR to add a regular member is approved when:
+
+* there are no outstanding objections
+* there are two or more approvals by voting CPC members
+* the PR has been opened for at least 1 week
+
+Once a PR is ready to be landed, the CPC member who lands the pull request should:
+
+* Send a notification to the project contacts for the project indentified in the PR
```

Comment:
Do individuals need to be representing a project as part of their CPC membership?

Reply:
The current governance says they must be part of one of the projects.  
>  Anyone who has been a member of one of the projects under the OpenJS Foundation for at least three months may request to become a regular member by opening a PR to add themselves to the list of regular members. 

https://github.com/openjs-foundation/cross-project-council/blob/master/CPC-CHARTER.md#regular-members

None of the text says they are "representing" the project although that is possibly implied.  The notification what we agreed to in terms of validating that they have been active in one of the projects for 3 months.